.github/scripts/register_runners.sh

```diff
@@ -1,15 +1,21 @@
 #!/bin/bash
-# SPDX-FileCopyrightText: 2022 Google LLC
+# SPDX-FileCopyrightText: 2022,2024 Google LLC
 #
 # SPDX-License-Identifier: Apache-2.0
 #
 # Usage:
 #
-#    register_runners <n_runners> <github_token>
+#    register_runners <n_runners> <labelA,labelB,...> <github_token>
 #
 # Example:
 #
-#    register_runners 8 jbg2PSMoQLfqvj0ZeBUW
+#    register_runners 8 self-hosted,compute jbg2PSMoQLfqvj0ZeBUW
```

Comment:
```suggestion
#    register_runners 8 compute jbg2PSMoQLfqvj0ZeBUW
```

`self-hosted` is applied automatically. I'm not sure what happens if you apply it manually too.

Reply:
Nonetheless, having an example on how to apply multiple labels is nice.